<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_page</class>
 <widget class="QWidget" name="Main_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="Left_posters">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>65</y>
     <width>250</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTableWidget" name="Rihgt_posters">
   <property name="geometry">
    <rect>
     <x>996</x>
     <y>65</y>
     <width>250</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Moviepoint</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>600</width>
     <height>300</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QListWidget" name="Movie_list">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="Schedule_list">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>311</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Buy_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>480</y>
     <width>180</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="movie_info_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>550</y>
     <width>180</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Movie info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="User_history_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>620</y>
     <width>180</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User history</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
